__init__.py

```diff
@@ -65,16 +65,16 @@ def execute(self, dict, key, value):
 class AssocImgNode:
     @classmethod
     def INPUT_TYPES(s):
-        return {"required": {"dict": ("DICT",), "key": ("STRING", {"default": ""}), "value": ("IMAGE", {"default": ""}), "format": (["webp", "png"], {"default": "webp"})}}
+        return {"required": {"dict": ("DICT",), "key": ("STRING", {"default": ""}), "value": ("IMAGE", {"default": ""}), "format": ("STRING", {"default": "webp"}), "quality": ("FLOAT", {"default": 92})}}
```

Comment:
added quality param and changed format to string. the defaults will be webp at a quality that's close to how cloudinary is storing our webps

Reply:
Do float defaults need to actually be floats and not ints? Probably not an issue, but figured I'd say something.